nautobot/extras/health_checks.py

```diff
@@ -160,5 +160,4 @@ def check_status(self):
         elif client_class == "django_redis.client.DefaultClient":
             self.check_redis(redis_url=location, **options.get("CONNECTION_POOL_KWARGS", {}))
         else:
-            if self.redis_url is None:
-                self.add_error(ServiceUnavailable(f"{client_class} is an unsupported CLIENT_CLASS!"))
+            self.add_error(ServiceUnavailable(f"{client_class} is an unsupported CLIENT_CLASS!"))
```

Comment:
Seems like an actual bug to me - I couldn't see anywhere `self.redis_url` was defined otherwise.